vignettes/tutorial-create-your-first-rhino-app.Rmd

```diff
@@ -7,11 +7,28 @@ vignette: >
   %\VignetteEncoding{UTF-8}
 ---
 
+<style>
```

Comment:
Let's move this to a separate CSS file which could be used on other pages as well - we should keep our styling consistent throughout articles.

Reply:
I am not sure the best way to do this with pkgdown. What would be the best approach in your mind?